themes/bootstrap3/templates/search/results.phtml

```diff
@@ -121,7 +121,7 @@
 
 <?php /* Refine Search Options */ ?>
 <?php if ($recommendations): ?>
-  <div class="<?=$this->layoutClass('sidebar')?>" id="search-sidebar">
+  <div class="<?=$this->layoutClass('sidebar')?>" id="search-sidebar" role="region" aria-label="<?=$this->transEscAttr('Search sidebar') ?>">
```

Comment:
I added some new translatable strings. Should I also add these to the translation files?

Reply:
Yes, please. You only need to add them to en.ini and any other languages you are fluent in; that will be enough to ensure that they get translated into the other languages by our volunteers on the next translation cycle.